mobile/src/wallets/cardano/api/fetch.ts

```diff
@@ -97,11 +97,10 @@ export const fetchDefault = <T = Record<string, any>>(
     checkResponse: options?.checkResponse ?? _checkResponse,
     headers,
   }
-  logger.debug(`fetchDefault: API call ${fullPath}`)
```

Comment:
bloating the logs, and is no longer needed, now with RN79 we can debug the network requests